icons/astro/astro-plain.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128px" height="128px" viewBox="0 0 128 128" version="1.1">
+<g id="surface1">
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(0%,0%,0%);fill-opacity:1;" d="M 81.503906 9.464844 C 82.476562 10.671875 82.972656 12.300781 83.960938 15.554688 L 105.617188 86.691406 C 97.609375 82.535156 88.90625 79.542969 79.726562 77.925781 L 65.628906 30.28125 C 65.398438 29.5 64.683594 28.964844 63.871094 28.964844 C 63.054688 28.964844 62.335938 29.503906 62.109375 30.285156 L 48.179688 77.902344 C 38.957031 79.511719 30.214844 82.511719 22.175781 86.679688 L 43.933594 15.539062 C 44.929688 12.289062 45.425781 10.664062 46.398438 9.457031 C 47.257812 8.394531 48.371094 7.570312 49.640625 7.058594 C 51.074219 6.484375 52.777344 6.484375 56.175781 6.484375 L 71.71875 6.484375 C 75.121094 6.484375 76.824219 6.484375 78.261719 7.0625 C 79.527344 7.570312 80.644531 8.398438 81.503906 9.464844 Z M 81.503906 9.464844 "/>
```

Comment:
Path's should be merged if possible, so you end up with a single path.
See the [SVG Standards](https://github.com/devicons/devicon/wiki/SVG-Standards) wiki page